<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmMain</class>
 <widget class="QMainWindow" name="FrmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FrmMain</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Anzahl Elemente</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edtAnzahlElemente"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>größte Zufallszahl</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edtGroussteZufallszahl"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QCheckBox" name="chBsortiert">
              <property name="text">
               <string>sortiert</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cBoxReihenfolgeInvertiert">
              <property name="text">
               <string>Reihenfolge Invertiert</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cBoxErsterWertAenderung">
              <property name="text">
               <string>erster Wert geändert auf:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="edtErsterWertGeaendert"/>
            </item>
            <item>
             <widget class="QPushButton" name="btnArrayErzeugen">
              <property name="text">
               <string>Array erzeugen</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Original</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="lwOriginal"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Kopie</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="lwKopie"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QPushButton" name="btnBubbleSort">
            <property name="text">
             <string>BubbleSort</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnBubbleSortOptim">
            <property name="text">
             <string>BubbleSort Optim.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSelectionSort">
            <property name="text">
             <string>SelecttionSort</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="text">
             <string>InsertSort</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnQuickSortGroening">
            <property name="text">
             <string>QuickSort Grüning</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnQuickSortSTL">
            <property name="text">
             <string>QuickSort STL</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLineEdit" name="edtTimeBubbleSort"/>
          </item>
          <item>
           <widget class="QLineEdit" name="edtBubbleSortOptim"/>
          </item>
          <item>
           <widget class="QLineEdit" name="edtSelectionSort"/>
          </item>
          <item>
           <widget class="QLineEdit" name="edtInsertSort"/>
          </item>
          <item>
           <widget class="QLineEdit" name="edtQuickSortGruening"/>
          </item>
          <item>
           <widget class="QLineEdit" name="edtQuickSortSTL"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
